Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/dao/ContactDiaryLocationDao.kt

```diff
@@ -0,0 +1,19 @@
+package de.rki.coronawarnapp.contactdiary.storage.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import de.rki.coronawarnapp.contactdiary.storage.entity.ContactDiaryLocationEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+abstract class ContactDiaryLocationDao : BaseRoomDao<ContactDiaryLocationEntity>() {
+
+    @Query("SELECT * FROM ContactDiaryLocationEntity")
```

Comment:
Is it uncommon for Android to push sorting down to the DB/SQL?

Reply:
No, depending on the data sizes, it's not really noticeable though. I'd prefer the solution that is easier to test.